src/amt/models.ts

```diff
@@ -352,3 +353,11 @@ export interface RemoteAccessPolicyAppliesToMPS extends CIM.Models.PolicySetAppl
   OrderOfAccess: number
   MpsType: number
 }
+
+export interface AlarmClockService extends CIM.Models.Service {
+  Name: string
```

Comment:
I think you can remove all the properties here, they exist in the types they extend

Reply:
I'll fix the duplicate variables today.  Thanks for the review.